server/index.js

```diff
@@ -10,24 +11,37 @@ app.use(express.json());
 app.use(express.static('public'));
 app.use(cors());
 
-//retrieve all products from database
-app.get('/items', (req, res) => {
-
-});
-
 //find product by id
 app.get('/items/id/:id', (req, res) => {
-  const { id } = req.params;
-
+  db.getById(req.params.id)
+    .then(results => res.send(results))
```

Comment:
Should you add a .catch in all of these in case of failure to see the error, it looks like you only did it for the last one?